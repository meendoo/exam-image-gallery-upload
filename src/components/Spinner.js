// Spinner taken from - https://loading.io/css/
import React from 'react'
import './spinner.scss'

export default function Spinner() {
  return (
    <div className="spinner-wrapper">
        <div className="lds-ring"><div></div><div></div><div></div><div></div></div>
    </div>
  )
}