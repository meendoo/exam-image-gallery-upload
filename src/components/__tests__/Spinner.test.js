import React from "react";
import Spinner from "../Spinner";
import renderer from "react-test-renderer";

describe("Spinner", () => {
  it("should render as it is", () => {
    const spinner = renderer.create(<Spinner />);
    expect(spinner).toMatchSnapshot();
  });
});
