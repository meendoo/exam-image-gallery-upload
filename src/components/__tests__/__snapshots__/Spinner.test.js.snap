// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner should render as it is 1`] = `
<div
  className="spinner-wrapper"
>
  <div
    className="lds-ring"
  >
    <div />
    <div />
    <div />
    <div />
  </div>
</div>
`;
